website/docs/getting-started/quickstart.mdx

```diff
@@ -74,6 +74,8 @@ Commands:
   taq init [projectDir]                  Initialize a new project
   taq scaffold [scaffoldUrl] [scaffoldP  Generate a new project using a pre
   rojectDir]                             -configured scaffold
+  taq optin                              Opt in for Google Analytics
+  taq optout                             Opt out of Google Analytics
```

Comment:
`Opt-out of sharing anonymous usage analytics`

Note: we will be using additional frameworks for collecting data so let's make it generalized by removing the reference to Google